.travis.yml

```diff
@@ -51,7 +51,7 @@ script:
   # Validate lua files
   - find $TRAVIS_BUILD_DIR -path $TRAVIS_BUILD_DIR/CorsixTH/Lua/languages -prune -o -name '*.lua' -print0 | xargs -0 luac -p --
   - cd $TRAVIS_BUILD_DIR
-  - /home/travis/.luarocks/bin/luacheck -q .
+  - /home/travis/.luarocks/bin/luacheck -q --no-max-line-length . --globals math.n_random --globals math.round --globals math.randomdump
```

Comment:
Shouldn't these settings (or at least the globals) go in the luacheckrc file with the other globals?